docs/user/vulnerabilities/vulnerability/product_status/known_not_affected-usage.en.md

```diff
@@ -1,12 +1,18 @@
 # Known Not Affected - Usage
 
-* A corresponding Full Product Name element with matching `product_id` must be defined.
+Lists only non-affected products.
 
-* A product listed here must not be listed in any of the following
+Products that are only not affected in a certain configuration must not be listed here.
```

Comment:
Add explanation:
they are considered affected as the user has to check the configuration (which is a remediation and can be decribed as action statement).